packages/ui/src/composables/useDropdownable.ts

```diff
@@ -0,0 +1,36 @@
+import { watch, ExtractPropTypes } from 'vue'
+import { useSyncProp } from './useSyncProp'
+import { extractComponentProps, filterComponentProps } from '../utils/component-options'
+import { VaDropdown } from '../components/va-dropdown'
+
+const VaDropdownProps = extractComponentProps(VaDropdown,
+  ['innerAnchorSelector', 'stateful', 'keyboardNavigation', 'modelValue', 'ariaLabel'],
+)
+
+export const useDropdownableProps = {
+  ...VaDropdownProps,
+  modelValue: {},
+  closeOnValueUpdate: { type: Boolean, default: false },
```

Comment:
I think this should be 3 states: `false`, `true`, `null`. 
* `false` - would never close on select
* `true` - close on select
* `null` - default component behavior

Here's couple of examples why this matters.

For select:
* single select should close on input by default
* multiselect should **not** close on input by default

For time picker
* when you have only one column - we need to close on input by default
* when you have multiple columns - we shoudn't close by default

Probably also applicable to date picker - where for single non range input we want to close by default.

Reply:
Also I read the user issue and it seems their concern was with defaults, not with ability to force closing on select.

Reply:
Defaults can be configured trough GlobalConfig.

Reply:
@m0ksem they can't be if you need different defaults depending on other props.